Tools/clinic/clinic.py

```diff
@@ -2951,11 +2938,27 @@ def parser_name(self):
 }
 
 
+ConverterDict = dict[str, CConverter]
+
+# maps strings to callables.
+# these callables must be of the form:
+#   def foo(name, default, *, ...)
+# The callable may have any number of keyword-only parameters.
+# The callable must return a CConverter object.
+# The callable should not call builtins.print.
+converters: ConverterDict = {}
+
+# maps strings to callables.
+# these callables follow the same rules as those for "converters" above.
+# note however that they will never be called with keyword-only parameters.
+legacy_converters: ConverterDict = {}
+
+
 class bool_converter(CConverter):
-    type = 'int'
+    type: str = 'int'
     default_type = bool
```

Comment:
If it's only ever `bool` or subclasses of `bool` on this class, instances of this class and subclasses of this class, use `bltns.type[bool]`. If it could be a different class on this class, instances of this class or subclasses of this class, use `bltns.type[Any]`. Or just use the same annotation from https://github.com/python/cpython/pull/104421/files#diff-4c6b55ec7d80a3d60f100ce230eb879d9acbe2e558e01d318960e6197d7c47b3R2592

Reply:
Ok, let me add those then.